### DISCUSSION TITLE
Fantasy Basketball H2H Categories Box Score

### DISCUSSION THREAD

**Discussion by JamesPHawk** [cwendt94/espn-api#390] [Q&A] [answered]
I'm in a Fantasy Basketball H2H Category League, which means the `box_score` class can't be used. I've been trying to adapt the `H2HCategoryBoxScore` code from the baseball api to use in the Basketball box_score file, but am not having much success. Anyone run into a similar issue?

**Comment by cwendt94** [comment, marked as answer]
Hey @JamesPHawk I have recently joined a H2H category league so that I can see how it is organized and update the basketball box_score with that functionality. I am going to try to get it done by next week!

**Reply by JamesPHawk** [reply]
Thank you, that would be awesome!

**Reply by cwendt94** [reply]
[PR](https://github.com/cwendt94/espn-api/pull/391) is almost ready, just need to add some tests. Should have it in tomorrow!

**Reply by cwendt94** [reply]
H2H Category is now in v0.26.0
